### PR TITLE
Switch to inherited stdout and stderr for udev tests

### PR DESCRIPTION
**Pull request by jbaublitz**
Related to #2724

### CODE REVIEW

**Comment by jbaublitz** [comment]
@mulkieran Putting up for review

File: tests/client-dbus/tests/udev/_utils.py [hunk 1]
```diff
@@ -300,12 +297,10 @@ def stop_service(self):
         :return: a tuple of stdout and stderr
         """
         self._service.send_signal(signal.SIGINT)
-        output = self._service.communicate()
+        self._service.wait()
         if list(processes("stratisd")) != []:
             raise RuntimeError("Failed to stop stratisd service")
 
-        return output
```
Comment (mulkieran) [inline tests/client-dbus/tests/udev/_utils.py:307]: Plz update docs for changed return type.

**Review by mulkieran — CHANGES_REQUESTED** [review]
One request, otherwise looks ready...

File: tests/client-dbus/tests/udev/_utils.py (hunk 1 above)
Comment (jbaublitz) [reply tests/client-dbus/tests/udev/_utils.py:307]: Resolved!

**Comment by mulkieran** [comment]
irrelevant failure, merging.

**Review by mulkieran — APPROVED** [review]

**Pull request merged**